charts/kubeclarity/templates/clusterrole.yaml

```diff
@@ -2,8 +2,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: {{ include "kubeclarity.name" . }}
-  labels:
-    {{ include "kubeclarity.labels" . }}
+  labels: {{ include "kubeclarity.labels" . | nindent 4 }}
```

Comment:
Hi @jmueller42 tnx for that.
I would prefer it will be put in place in a different PR and not along with the service monitor stuff.
Also, please, can you keep the format of
```
  labels:
    {{- include "kubeclarity.labels" . | nindent 4 }}
```

Reply:
okay

Reply:
> I would prefer it will be put in place in a different PR and not along with the service monitor stuff.

@jmueller42  can you create 2 different PRs, 1 improving the `kubeclarity.labels` and the other one for the servicemonitor stuff?

Reply:
@FrimIdan The problem is the labels need to be indented 4 spaces in most of the templates but 6 spaces in the ServiceMonitor template

Reply:
Not sure I understand how the indent is related.
I just asked to create 2 PRs
1. Improves the helper function: "kubeclarity.labels" in _helpers.tpl
2. Add servicemonitor

we will merge them one after the other.

Reply:
okay i will prepare 2 PRs

Reply:
@FrimIdan I created the first PR: https://github.com/openclarity/kubeclarity/pull/454

When this is merged i will update this one.

Reply:
@FrimIdan I updated this PR with the other changes